### ISSUE TITLE
Better column summary descriptions

### ISSUE THREAD

**Issue by yochannah** [intermine/imjs#16]
Using the FlyMine
   Organism -> transcripts
template and the column summary on transcript length I realise the column summary
is doing something subtley different from what one might expect:

 - while there are 34,428 rows it reports that there are 7538 transcript lengths.

This should be changed to "7538 distinct transcript lengths" or perhaps better:

"7538 distinct transcript lengths for 34,428 transcripts"

**Comment by CodHeK** [comment]
@yochannah I can try solving this!

**Comment by yochannah** [comment]
Sure, go ahead!

**Comment by CodHeK** [comment]
If you could provide a little more briefing on this ... the example query that you tried ...

**Comment by yochannah** [comment]
sure! We've done another data build since the problem was reported, so the numbers are slight different from what I wrote in June, but they're still close. 

1. go to http://www.flymine.org/ and click on templates. 
2. Look for the `Organism  -->  All genes and their transcripts, exons and introns` template. Go to it, then click the green "Show results" button.
3. On the results screen, take a note of the number of rows in the table. Around 30k.
3. Click on the little barchart above the Transcripts length column. It tells you there are around 7k transcript lengths. 

Soooo, one might wonder - which is it? 7k or 30k? Hence the suggestion above for re-wording it to make things clearer. 

Let me know if that makes sense!

**Comment by CodHeK** [comment]
Yeah makes sense, we can just add the word `unique` I guess, makes it pretty clear! `Showing distribution of 7k unique Transcript lengths` ?

**Comment by yochannah** [comment]
different column types have different displays depending on the data inside them - but for every column of this type, this description should be modified :) 

Let's stick with distinct rather than unique, though, I think.

**Comment by CodHeK** [comment]
Shouldn't this be a `im-tables` issue :p

**Comment by yochannah** [comment]
LOL. Yes!

um, I was just testing you?
